d2l-button-shared-styles.html

```diff
@@ -1,7 +1,14 @@
 <link rel="import" href="../polymer/polymer.html">
 <style is="custom-style">
 	html {
+		--d2l-button-clear-focus: {
+			box-shadow: 0 0 0 4px rgba(0, 0, 0, 0);
+		};
+		--d2l-button-focus: {
+			box-shadow: 0 0 0 4px var(--d2l-color-celestine-light);
+		};
 		--d2l-button-spacing: 0.75rem;
+		--d2l-color-celestine-light: rgba(0, 111, 191, 0.3);
```

Comment:
the only thing is that we'll re-use the `--d2l-button-focus` for other buttons (i.e. `button-menu`), if we copy `--d2l-button-focus` code everywhere (instead of centralize it), it will be so error-prone if we change the foucs style later (remembering all places again). We definitely can move `--d2l-color-celestine-light`  inside the `--d2l-button-focus` if that speed things up.